packages/background/src/index.integration.test.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable jest/no-commented-out-tests */
```

Comment:
no need to add this line since there's no commented out tests
we can use test.todo(...) as comment

Reply:
ok, have already removed this line

Reply:
> better use the `jest.config.js` as a baseConfig and apply changes to it for integration tests

yes, have already extracted the integration config based on `jest.config.js`